test/from_handlers_test.dart

```diff
@@ -136,18 +136,23 @@ void main() {
     group('broadcast stream with multiple listeners', () {
       int dataCallCount;
       int doneCallCount;
+      int errorCallCount;
 
       setUp(() async {
         dataCallCount = 0;
         doneCallCount = 0;
+        errorCallCount = 0;
         setUpForController(
             new StreamController.broadcast(),
             fromHandlers(handleData: (value, sink) {
               dataCallCount++;
+            }, handleError: (error, stackTrace, sink) {
+              errorCallCount++;
+              sink.addError(error, stackTrace);
             }, handleDone: (sink) {
               doneCallCount++;
             }));
-        transformed.listen((_) {});
+        transformed.listen((_) {}, onError: (_, __) {});
```

Comment:
When there are multiple unused arguments in the signature I use 1 underscore for the first argument, 2 for the next, etc. Luckily this doesn't come up often.